app/assets/stylesheets/multi_page.scss

```diff
@@ -0,0 +1,27 @@
+$base        : #AAAAAA;
+$secondary-color: white;
+
+.flash {
+  background-color: maroon;
```

Comment:
Color `maroon` should be written in hexadecimal form as `#800000`<br>Color literals like `maroon` should only be used in variable declarations; they should be referred to via variable everywhere else.